tests/integration/components/bs-tooltip-test.js

```diff
@@ -342,19 +342,6 @@ module('Integration | Component | bs-tooltip', function (hooks) {
     assert.ok(isVisible(this.element.querySelector('.tooltip')), '200ms: tooltip is faded in');
   });
 
-  test('should not show tooltip if leave event occurs before delay expires', async function (assert) {
```

Comment:
Is this test removal somehow related to the commit intention: `manually fix remaining eslint errors`?